spark-3/src/main/scala/org/neo4j/spark/reader/Neo4jScanBuilder.scala

```diff
@@ -126,4 +126,7 @@ class Neo4jScanBuilder(neo4jOptions: Neo4jOptions, jobId: String, schema: Struct
     topN = Some(TopN(limit, orders))
     true
   }
+
+  // otherwise doesn't compile in Scala 2.12
+  override def isPartiallyPushed: Boolean = true
```

Comment:
so the interface is different between spark for Scala 2.12 and 2.13??

Reply:
I don't think so, I just get this error by the compiler in Intellj:

```
class Neo4jScanBuilder inherits conflicting members:
  method isPartiallyPushed in trait SupportsPushDownLimit of type ()Boolean  and
  method isPartiallyPushed in trait SupportsPushDownTopN of type ()Boolean
(Note: this can be resolved by declaring an override in class Neo4jScanBuilder.)
class Neo4jScanBuilder(neo4jOptions: Neo4jOptions, jobId: String, schema: StructType)
```

Reply:
As a side note, both of them are default methods that return `true`.